medicines/search-client/src/models.rs

```diff
@@ -1,14 +1,16 @@
+use chrono::{SecondsFormat, Utc};
 use core::fmt::Debug;
 use serde_derive::{Deserialize, Serialize};
+use std::clone::Clone;
 
-#[derive(Debug, Deserialize)]
+#[derive(Clone, Debug, Deserialize)]
 pub struct AzureHighlight {
     #[serde(rename = "content")]
     content: Vec<String>,
 }
 
-#[derive(Debug, Deserialize)]
-pub struct AzureResult {
+#[derive(Clone, Debug, Deserialize)]
+pub struct IndexResult {
```

Comment:
not sure there's such a big benefit in avoiding the `Azure` name if `highlights` has an `AzureHighlight` type and all the fields are married to Azure's naming

Reply:
We do at least control the fields in `IndexResult` and `IndexEntry` since we define the shape of the index. The `AzureHighlight` I think is dictacted by Azure.